velox/dwio/parquet/tests/reader/ParquetReaderTest.cpp

```diff
@@ -103,3 +120,95 @@ TEST_F(ParquetReaderTest, parseRowMapArray) {
   auto col0_1_1_0 = col0_1_1->childAt(0);
   EXPECT_EQ(col0_1_1_0->type->kind(), TypeKind::INTEGER);
 }
+
+TEST_F(ParquetReaderTest, varcharFilters) {
+  // name < 'CANADA'
+  FilterMap filters;
+  filters.insert({"name", exec::lessThan("CANADA")});
+
+  auto expected = vectorMaker_->rowVector({
+      vectorMaker_->flatVector<int64_t>({0, 1, 2}),
```

Comment:
Use VectorTestBase as a base class to simplify this code to

`makeFlatVector<int64_t>({0, 1, 2})`